neo/Consensus/ConsensusService.cs

```diff
@@ -261,11 +262,24 @@ private void OnConsensusPayload(ConsensusPayload payload)
                 return;
             }
             if (payload.ValidatorIndex >= context.Validators.Length) return;
+            ConsensusMessage message;
+            try
+            {
+                message = payload.ConsensusMessage;
+            }
+            catch (FormatException)
```

Comment:
I prefer to catch generic exceptions

Reply:
Try not to catch generic exceptions, or the future debugging work will become extremely difficult.

Reply:
Erik, if this may fail, we should enforce somewhere else too, no?

Reply:
Erik, if this may fail, we should enforce somewhere else too, no?